lib/instrumentation/modules/koa.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+var semver = require('semver')
+
+module.exports = function (koa, agent, version, enabled) {
+  if (!enabled) return koa
+
+  if (!agent._conf.frameworkName) agent._conf.frameworkName = 'koa'
+  if (!agent._conf.frameworkVersion) agent._conf.frameworkVersion = version
+
+  if (!semver.satisfies(version, '^2.0.0')) {
```

Comment:
Since we don't need to do anything with the Koa module it self, there's no reason to have this if-block validating the version. Similarly you can also just remove the `require('semver')` at the top.

Reply:
I did! thanks :)